detect/errorloaderdetector.py

```diff
@@ -0,0 +1,36 @@
+import pandas as pd
+
+from .detector import Detector
+
+
+class ErrorsLoaderDetector(Detector):
+    """
+    Detector that loads a table of constant errors from csv with the columns:
+        _tid_: entity ID
+        attribute: attribute in violation
+        in the format _tid_, attribute
+    """
+    def __init__(self, fpath, name='ErrorLoaderDetector'):
+        """
+        :param name: (str) name of the detector
+        :param fpath: (str) path to csv file to load errors from.
+        """
+        super(ErrorsLoaderDetector, self).__init__(name)
+        self.errors_df = pd.read_csv(fpath, dtype=str, encoding='utf-8')
+        if list(self.errors_df) != ['_tid_', 'attribute']:
```

Comment:
shouldn't it be `self.errors_df.columns.tolist()`? This retrieves the tuples, not the columns

Reply:
This indeed retrieves the columns not tuples.
Please see: https://stackoverflow.com/questions/19482970/get-list-from-pandas-dataframe-column-headers

Reply:
SG